elastic/kibana
# [ObsUX][APM] Replace `react-syntax-highlighter` with `EuiCodeBlock`

## Summary

Removes `react-syntax-highlighter` from APM errors, in favour of `EuiCodeBlock` for read-only code syntax highlighting. This in turn removes a bunch of custom styling to bring things more inline with the design system as well.

Closes #204049

## How to test

* Go to Applications - Service Inventory
* Find a service with errors
* Go to Errors tab for service
* Select an error that is an exception
* View details for the exception and see the syntax highlighted block for the stack trace.
